src/client/swagger/unity.yml

```diff
@@ -905,6 +934,47 @@ components:
               type: string
               format: uri
       required: [id, email, role]
+    Me:
+      properties:
+        id:
+          description: the idpe id of the user
+          type: string
+        firstName:
+          type: string
+        lastName:
+          type: string
+        email:
+          type: string
+        orgId:
+          type: string
```

Comment:
I'm torn about whether we need `id` and `orgId` in this payload since we have them in the session already. But it's probably easier in the UI to have them here. In that case, should we also include `accountId` so we have a full set?

Reply:
We don't necessarily need the orgID here since we can obtain that through other means, I just wasn't sure about where / when the API would be used so I'm trying to keep it as dynamic as possible. Good call on the accountId